src/odemis/driver/test/xt_client_test.py

```diff
@@ -226,9 +260,10 @@ def test_vent_and_pump(self):
         self.microscope.pump()
         self.assertEqual(self.microscope.get_vacuum_state(), 'vacuum')
```

Comment:
These vacuum related function should be accessed via an "Chamber" actuator. See the Phenom driver API for an example.